src/custom-elements/src/components/toggle-switch/toggle-switch.tsx

```diff
@@ -26,6 +26,11 @@ export class ToggleSwitch {
    */
   @Prop({ reflect: true, mutable: true }) checked = false;
 
+  /**
+   * OPTIONAL. Size of the label. {xs,sm}.
+   */
+  @Prop({ reflect: true }) size: string;
```

Comment:
```suggestion
  @Prop({ reflect: true }) size?: string;
```
Let's add an interrogation symbol here to define the prop as optional.
We should do the same for all the optional props of other web components as well